src/include/txn.h

```diff
@@ -334,26 +347,27 @@ struct __wt_txn {
  */
 
 /* AUTOMATIC FLAG VALUE GENERATION START 0 */
-#define WT_TXN_AUTOCOMMIT 0x00001u
-#define WT_TXN_ERROR 0x00002u
-#define WT_TXN_HAS_ID 0x00004u
-#define WT_TXN_HAS_SNAPSHOT 0x00008u
-#define WT_TXN_HAS_TS_COMMIT 0x00010u
-#define WT_TXN_HAS_TS_DURABLE 0x00020u
-#define WT_TXN_HAS_TS_PREPARE 0x00040u
-#define WT_TXN_IGNORE_PREPARE 0x00080u
-#define WT_TXN_IS_CHECKPOINT 0x00100u
-#define WT_TXN_PREPARE 0x00200u
-#define WT_TXN_PREPARE_IGNORE_API_CHECK 0x00400u
-#define WT_TXN_READONLY 0x00800u
-#define WT_TXN_RUNNING 0x01000u
-#define WT_TXN_SHARED_TS_DURABLE 0x02000u
-#define WT_TXN_SHARED_TS_READ 0x04000u
-#define WT_TXN_SYNC_SET 0x08000u
-#define WT_TXN_TS_NOT_SET 0x10000u
-#define WT_TXN_TS_ROUND_PREPARED 0x20000u
-#define WT_TXN_TS_ROUND_READ 0x40000u
-#define WT_TXN_UPDATE 0x80000u
+#define WT_TXN_AUTOCOMMIT 0x000001u
+#define WT_TXN_ERROR 0x000002u
+#define WT_TXN_HAS_ID 0x000004u
+#define WT_TXN_HAS_SNAPSHOT 0x000008u
+#define WT_TXN_HAS_TS_COMMIT 0x000010u
+#define WT_TXN_HAS_TS_DURABLE 0x000020u
+#define WT_TXN_HAS_TS_PREPARE 0x000040u
+#define WT_TXN_IGNORE_PREPARE 0x000080u
+#define WT_TXN_IS_CHECKPOINT 0x000100u
+#define WT_TXN_PREPARE 0x000200u
+#define WT_TXN_PREPARE_IGNORE_API_CHECK 0x000400u
+#define WT_TXN_READONLY 0x000800u
+#define WT_TXN_REFRESH_SNAPSHOT 0x001000u
```

Comment:
Can we add eviction to the name?

Reply:
As discussed offline - not doing this one.